features/include/pcl/features/impl/feature.hpp

```diff
@@ -334,5 +329,6 @@ pcl::FeatureWithLocalReferenceFrames<PointInT, PointRFT>::initLocalReferenceFram
   return (true);
 }
 
-#endif  //#ifndef PCL_FEATURES_IMPL_FEATURE_H_
+} // namespace pcl
 
+#endif  //#ifndef PCL_FEATURES_IMPL_FEATURE_H_
```

Comment:
Some files seem to be missing a new line at the end.

PS: I'm pointing out `pragma once` because the lines got touched and I happened to see them. Leaving this for a different PR would also be ok.